great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -32,6 +44,33 @@ class ResponsePayload(TypedDict):
     data: PayloadDataField
 
 
+AnyPayload = Union[ResponsePayload, ErrorPayload]
+
+
+def _get_user_friendly_error_message(
+    http_exc: requests.exceptions.HTTPError,
+) -> str:
+    # TODO: define a GeCloud service/client for this & other related behavior
+    support_message = []
+    response: requests.Response = http_exc.response
+
+    logger.info(f"{http_exc.__class__.__name__}:{http_exc} - {response}")
+
+    request_id = response.headers.get("request-id", "")
+    if request_id:
+        support_message.append(f"Request-Id: {request_id}")
+
+    try:
+        error_json: ErrorPayload = http_exc.response.json()
+        errors = error_json.get("errors")
+        if errors:
+            support_message.append(json.dumps(errors))
+
+    except json.JSONDecodeError:
+        support_message.append(f"Please contact superconductive at {SUPPORT_EMAIL}")
```

Comment:
Note that this is only provided if there is no JSON content in the response for us to include.
For example, if GE Cloud has an outage.